mitosheet/src/mito/components/endo/FormulaBar.tsx

```diff
@@ -35,10 +35,10 @@ const FormulaBar = (props: {
 
     const rowIndex = props.selection.startingRowIndex
     const colIndex = props.selection.startingColumnIndex
-
-    const {columnHeader, columnFormula, cellValue, columnFormulaLocation} = getCellDataFromCellIndexes(props.sheetData, rowIndex, colIndex);
+    const sheetData = props.sheetDataArray[props.sheetIndex];
```

Comment:
Shouldn't we be using the editorState.sheetIndex here? I think we actually probably are -- but in that case, let's remove the sheetIndex prop from the FormulaBar and just use the editorState.sheetIndex everywhere in this file for clarify. Otherwise, we have two variables confusingly referring to the same thing.